### PR TITLE
[CLOUD-2221] configure KieLoginModule only for Business Central

### PR DESCRIPTION
**Pull request by psiroky**
Thanks for submitting your Pull Request!

Please make sure your PR meets following requirements:

- [x] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull request does not include fixes for other issues than the main ticket
- [x] Attached commits represent unit of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@redhat.com>` - use `git commit -s`

This is definitely not an ideal solution. We should probably have generic set of common functions/modules and then specify which of them would be executed for the specific deliverables. But until that is in place this should be a sufficient workaround.

### CODE REVIEW

**Comment by psiroky** [comment]
CC @goldmann

**Comment by psiroky** [comment]
@spolti could you please take a look and review this?

**Comment by spolti** [comment]
@psiroky  wouldn't businesscentral-monitoring need this too?

**Comment by psiroky** [comment]
@spolti good point about the businesscentral-monitoring - that will likely need it too (or at least it was the current behaviour, so better leave it as is).

I don't see any bmpsuite tests (yay) to start from, but will take a look at adding some based on tests from other modules.

**Comment by spolti** [comment]
I created a jira to add those tests: CLOUD-2213
I added a couple of basic tests, @goldmann @psiroky do you think we can get the tests merged?

**Comment by psiroky** [comment]
I have approved the PRs for CLOUD-2213. I am afraid I can't do much more at this point.

**Comment by psiroky** [comment]
@spolti tests added. Could you please check again?

**Comment by spolti** [comment]
You added tests for rhdm but in the scripts you check only for businesscentral, is this configuration needed on decision manager too?

**Comment by psiroky** [comment]
@spolti the tests are making sure the custom login-module is _not_ configured as it actually breaks the Decision Central (see the linked JIRA).

File: tests/features/bpmsuite/businesscentral/configuration.feature [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@jboss-bpmsuite-7/bpmsuite70-businesscentral-openshift @jboss-bpmsuite-7/bpmsuite70-businesscentral-monitoring-openshift
+Feature: Business Central configuration tests
+
+  # https://issues.jboss.org/browse/CLOUD-2221
+  Scenario: Check KieLoginModule is configured
+  Given XML namespaces
+        | prefix | url                           |
```
Comment (spolti) [inline tests/features/bpmsuite/businesscentral/configuration.feature]: It is not necessary as you aren't using xml namespace on the tests.

File: tests/features/rhdm/decisioncentral/configuration.feature [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@rhdm-7/rhdm70-decisioncentral-openshift
+Feature: Decision Central configuration tests
+
+  # https://issues.jboss.org/browse/CLOUD-2221
+  Scenario: Check KieLoginModule is _not_ configured
+  Given XML namespaces
```
Comment (spolti) [inline tests/features/rhdm/decisioncentral/configuration.feature]: same here.

File: tests/features/bpmsuite/businesscentral/configuration.feature (hunk 1 above)
Comment (psiroky) [reply tests/features/bpmsuite/businesscentral/configuration.feature]: Ah, good point, thanks. I was trying to use the xpath way, but that's actually not necessary for this simple case. Will remove that.

**Pull request merged**